How-to-upgrade-review-version.md

```diff
@@ -0,0 +1,399 @@
+# How to Upgrade to TypeScript Without Anybody Noticing
+
+This guide will show you how to upgrade to Typescript without anybody
+noticing. Well, people *might* notice &mdash; what I really mean is
+that you won't have to change your build at all. You'll have the
+ability to get errors and completions in your editor and run `tsc`
+from the command line, but you won't have to integrate Typescript into
+your build.
+
+The way to do this is to not use Typescript. Stick with Javascript and
+use JSDoc to provide type information. This is less convenient than
+Typescript's syntax for types, but it means that your files stay
+boring old Javascript.
+
+Here's how. I'm going to use
+[typescript-eslint-parser](https://github.com/eslint/typescript-eslint-parser)
+as an example package so you can follow along if you want. Notably,
+this package *parses* Typescript, but is *written* in Javascript.
+
+## Add tsconfig
```

Comment:
Why would I prefer this approach over other methods that immediately spring to mind such as 
- using `checkjs`
- adding a `jsconfig.json`
- relying on automatic type acquisition in VS Code

Each are slightly different but the pros and cons would be nice to call out.

Reply:
It the document is aimed to TS novices, it might me too much information to process. Instead maybe mentioned that they are other methods not described here and post a link to further documentation?

Reply:
Good idea @armanio123. I think this deserves another document. Unfortunately I couldn't find one, so I think it needs to be added.

I believe the place it should go is on the Quick Start page, right next to the "Convert project to TS" link.